Clarify that register_new_matrix_user is present also when installed via non-pip package (#9074)
matrix-org/synapse: INSTALL.md

The easiest way to create a new user is to do so from a client like [Element](https://element.io/).

Alternatively, you can do so from the command line. This can be done as follows:

> ```sh
> $ # These steps are needed if synapse was installed via pip.
> $ source ~/synapse/env/bin/activate
> $ synctl start # if not already running
> $
> $ # If synapse was installed via a prebuilt package, register_new_matrix_user
> $ # should already be on the search path.
> $ register_new_matrix_user -c homeserver.yaml http://localhost:8008

Reviewer: ```suggestion
 1. If synapse was installed via pip, set up the virtualenv as follows (if synapse was
    installed via a prebuilt package, `register_new_matrix_user` should already be
    on the search path):
    ```sh
    cd ~/synapse
    source env/bin/activate
    synctl start # if not already running
    ```
 2. ```sh
    register_new_matrix_user -c homeserver.yaml http://localhost:8008
```

The easiest way to create a new user is to do so from a client like [Element](https://element.io/).

> Alternatively, you can do so from the command line if you have installed synapse via package.
> 
> This can be done as follows:

Reviewer: This doesn't really make much sense as it is. 

```suggestion
Alternatively, you can do so from the command line. This can be done as follows:
```

```sh

> $ # These steps are needed if installed via pip.

Reviewer: ```suggestion
$ # These steps are needed if you installed via pip.
```

Author: Hey, I think it's better for docs to be imperative or passive, hope you're satisfied with the update

$ source ~/synapse/env/bin/activate
$ synctl start # if not already running
$

> $ # This executable is present when installed via any package.

Reviewer: ```suggestion
$ # If you installed via a prebuilt package, register_new_matrix_user should already be on
$ # the search path.
```
